README.md

```diff
@@ -1,54 +1,53 @@
 # Fedora Cosmic Atomic
-### Fedora Silverblue-based Atomic Desktop with the Pre-Alpha Cosmic Desktop Environment Included!
+### Fedora Silverblue-based Atomic Desktop with the Pre-Alpha Cosmic Desktop Environment included!
```

Comment:
I prefer the capitalization here actually.

Reply:
the header is not capitalized so I dont see the point? I am not english native so no idea how this is done normally